scripts/newinstall.sh

```diff
@@ -45,6 +45,8 @@ help_flag=false
 noop_flag=false
 
 # Use system python to bootstrap unless otherwise specified
+# This is used to install and run EUPS and will not necessarily
+# be the python in the path being used to build the stack itself.
 PYTHON="${PYTHON:-/usr/bin/python}"
```

Comment:
What about changing this to something along the lines of `PYTHON="${PYTHON:-$(which python)}"` then consistently using `$PYTHON` through out this script?  I thinking testing the path resolved `python` version to decide whether or not to install miniconda but eups always being installed with `--with-python="$PYTHON"` really strange behavior.